### PR TITLE
Update sys-dm-os-wait-stats-transact-sql.md

### PR DESCRIPTION
**Pull request by asavioli**
Adding description for the new wait BUFFERPOOL_SCAN, from new feature Buffer Pool Parallel Scan

### CODE REVIEW

File: docs/relational-databases/system-dynamic-management-views/sys-dm-os-wait-stats-transact-sql.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ GO
...
 |<a name="broker_transmitter"></a>BROKER_TRANSMITTER |Occurs when the Service Broker transmitter is waiting for work. Service Broker has a component known as the Transmitter which schedules messages from multiple dialogs to be sent across the wire over one or more connection endpoints. The transmitter has 2 dedicated threads for this purpose. This wait type is charged when these transmitter threads are waiting for dialog messages to be sent using the transport connections. High values of `waiting_tasks_count` for this wait type point to intermittent work for these transmitter threads and are not indications of any performance problem. If service broker is not used at all, `waiting_tasks_count` should be 2 (for the 2 transmitter threads) and wait_time_ms should be twice the duration since instance startup. See [Service broker wait stats](/archive/blogs/sql_service_broker/service-broker-wait-types).|
+|BUFFERPOOL_SCAN | May occur when the buffer pool scan runs in parallel and the main task waits for the scan to complete . <br /><br /> **Applies to**: [!INCLUDE[ssSQL22](../../includes/sssql22-md.md)] and later.|
```
Comment (PiJoCoder) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-os-wait-stats-transact-sql.md:132]:
```suggestion
|BUFFERPOOL_SCAN | May occur when the buffer pool scan runs in parallel and the main task waits for the scan to complete . For more information see [Operations that trigger a buffer pool scan may run slowly on large-memory computers](/troubleshoot/sql/performance/buffer-pool-scan-runs-slowly-large-memory-machines) <br /><br /> **Applies to**: [!INCLUDE[ssSQL22](../../includes/sssql22-md.md)] and later.| 
```

Comment (rwestMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-os-wait-stats-transact-sql.md:132]:
```suggestion
|BUFFERPOOL_SCAN | May occur when the buffer pool scan runs in parallel and the main task waits for the scan to complete. <br /><br /> **Applies to**: [!INCLUDE[ssSQL22](../../includes/sssql22-md.md)] and later. | 
```

**Comment by jborsecnik** [comment]
@westMSFT, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@jborsecnik I have already reviewed the proposed changes, and am awaiting response to my suggestions.

**Comment by asavioli** [comment]
#sign-off

**Comment by rwestMSFT** [comment]
This PR will be replaced by a new one because I also did some article refreshment.

**Pull request closed without merging**